src/lib/tuning/greedy_selector.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "tuning/abstract_selector.hpp"
+#include "tuning/tuning_choice.hpp"
+#include "tuning/tuning_operation.hpp"
+
+namespace opossum {
+
+/**
+ * The GreedySelector uses the following algorithm to determine TuningOperations:
+ * 1. Sort Tuning Choices by ascending desirability, within same desirability by ascending cost.
+ *    Then repeatedly performs the following operations:
+ * 2. Tries to free up budget by rejecting choices with lowest desirabilities ("starting from the end of list")
+ * 3. Tries to fill unused budget by accepting choices with highest desirabilities ("starting from the start of the list")
```

Comment:
The documentation of this could *really* benefit from a short example

Reply:
We added a link to the example in our slides.